user-guide/network/nictag.rst

```diff
@@ -11,18 +11,34 @@ For other use cases, it is possible to add following NIC tags:
 - internal
 - storage
 
-.. note:: **external**, **internal**, **storage** NIC tags are preconfigured in the *Danube Cloud* system. However, it is up to the data center administrator to do the proper configuration on the compute nodes.
+Administrator can also add new NIC tags with custom name or type (normal, overlay or etherstub). These new NIC tags are then collected from the node when Refresh button in node's Details view is pressed.
```

Comment:
Please rephrase this and add a link to node_actions tag. Do not specify the GUI path here, instead link to the appropriate section.
Example of good phrasing:
> `After manually configuring storage settings directly on the compute node, the :ref:`compute node's system information must be refreshed <node_actions>` in the *Danube Cloud* web management.`